en_US/README.md

```diff
@@ -1,21 +1,22 @@
 # Introduction (5.0 doc is under construction)
 
-*EMQX* is an open source MQTT message broker based on the Erlang/OTP platform.
+*EMQX* is an Open-Source, Cloud-Native, Distributed MQTT Broker for IoT based on the Erlang/OTP platform.
 
 Erlang/OTP is an excellent Soft-Realtime, Low-Latency and Distributed development platform.
 
 MQTT is a lightweight message exchange protocol using publish-subscribe pattern.
 
-*EMQX* is designed for massive clients access and realizes fast and low-latency message routing between massive physical network devices:
+*EMQX* is the world's most scalable distributed MQTT broker with a high-performance real-time message processing engine, powering event streaming for IoT devices at massive scale.
```

Comment:
Do we have any possible links to something showing it is the most scalable broker that we could inject?

Reply:
The 'most' is a very controversial word. Change to "highly scalable distributed MQTT Broker"?